kernels/transform_matmul/matmul.transform.mlir

```diff
@@ -0,0 +1,19 @@
+ func.func @simple_matmul(%arg0: memref<16x16xi8>, %arg1: memref<16x16xi8, strided<[1, 16], offset:0>>, %arg2: memref<16x16xi32>) {
+    %c0_i32 = arith.constant 0 : i32
+    linalg.quantized_matmul ins(%arg0, %arg1, %c0_i32, %c0_i32 : memref<16x16xi8>, memref<16x16xi8, strided<[1, 16], offset:0>>, i32, i32) outs(%arg2 : memref<16x16xi32>)
+    return
+ }
+
+
+transform.sequence failures(propagate) {
+^bb0(%arg0: !transform.any_op,
+     %arg1: !transform.op<"linalg.quantized_matmul">):
+  // The actual tiling transformation takes tile sizes as attributes.
+   %loop1, %loop2, %tiled = transform.structured.tile %arg1 [8, 8]
+    : (!transform.op<"linalg.quantized_matmul">) -> (!transform.any_op, !transform.any_op, !transform.any_op)
+  transform.yield
+}
+
```

Comment:
If this is the only line that is changing alongside the added options to run the compiler, we might just want to add this to another folder instead, of adding yet another entire 200 lines. @jorendumoulin what do you think?